build/infrastructure/eo/chart/Chart.yaml

```
@@ -3,8 +3,4 @@ name: eo-frontend
 description: A Helm chart for Kubernetes
 
 type: application
-version: 0.2.117
-dependencies:
-  - name: eo-base-helm-chart
-    version: 'v0.4.7'
-    repository: 'https://energinet-datahub.github.io/helm-charts/'
+version: 0.2.118
```

```suggestion
version: 0.3.0
```
Larger changes to chart should signify major or minor version bump